pyquil/tests/test_latex.py

```diff
@@ -1,22 +1,63 @@
-from pyquil.quil import Program
-from pyquil.gates import X, Y, CZ, SWAP, MEASURE, CNOT
-from pyquil.latex import to_latex
-from pyquil.latex.latex_config import get_default_settings
+from pyquil.quil import Program, Pragma
+from pyquil.gates import X, Y, RX, CZ, SWAP, MEASURE, CNOT
+from pyquil.latex import to_latex, DiagramSettings
+
+import pytest
```

Comment:
lets's move the entire `test_latex.py` file into a `tests` directory in the `latex` module